<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileManager</class>
 <widget class="QWidget" name="FileManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>213</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>213</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>213</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QListView" name="ui_directoryBrowser">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ui_mountButton">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string>QPushButton{
border:0px;
background-image:url(:/actions/rivet80x30L.png);
color:black;
}
QPushButton:pressed{
border:0px;
background-image:url(:/actions/rivet80x30D.png);
color:white;
}</string>
       </property>
       <property name="text">
        <string>Mount USB</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ui_unmountButton">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string>QPushButton{
border:0px;
background-image:url(:/actions/rivet80x30L.png);
color:black;
}
QPushButton:pressed{
border:0px;
background-image:url(:/actions/rivet80x30D.png);
color:white;
}</string>
       </property>
       <property name="text">
        <string>Unmnt. USB</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ui_actionButton">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border:0px;
background-image:url(:/actions/rivet80x30L.png);
color:black;
}
QPushButton:pressed{
border:0px;
background-image:url(:/actions/rivet80x30D.png);
color:white;
}</string>
       </property>
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ui_stopButton">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border:0px;
background-image:url(:/actions/rivet80x30L.png);
color:black;
}
QPushButton:pressed{
border:0px;
background-image:url(:/actions/rivet80x30D.png);
color:white;
}</string>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ui_deleteButton">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string>QPushButton{
border:0px;
background-image:url(:/actions/rivet80x30L.png);
color:black;
}
QPushButton:pressed{
border:0px;
background-image:url(:/actions/rivet80x30D.png);
color:white;
}</string>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
